codegen/test/rtc/src/compile_kernel.cpp

```diff
@@ -1,10 +1,16 @@
 #include "rtc/hip.hpp"
 #include <rtc/compile_kernel.hpp>
+// TODO include only if USE_RTC is set?
+#include <hip/hiprtc.h>
```

Comment:
We should use an `ifdef` instead of env variable, and that ifdef can be added based on a cmake variable.